wit_dashboard/wit-attribution-table.html

```diff
@@ -71,17 +80,28 @@
       properties: {
         attributions: Array,
         models: Array,
+        colors: Object,
+        textColorFunction: Object,
       },
       displayNum: function(num) {
         return d3.format('.3f')(num);
       },
       displayAttributionHeader: function(models, index) {
-        let str = 'attribution';
         if (models != null && models.length > 1) {
           const modelName = models[index];
-          str += ` (model ${modelName})`;
+          return `Model ${modelName}`;
+        } else {
+          return 'Attribution';
+        }
+      },
+      getStyle: function(num) {
+        if (this.colors == null) {
+          return '';
         }
-        return str;
+        const domain = this.colors.domain();
+        return 'background-color:' + this.colors(num) +
```

Comment:
getColorForSaliency [in example-viewer](https://github.com/PAIR-code/what-if-tool/blob/master/wit_dashboard/wit-example-viewer.ts#L1942) seems to have some sort of thresholding for colors through showSaliencyForValue. Here you call colors(num) directly. It would be nice to get background color from a single place as well like you did with text color. I leave this up to you.

Reply:
ah that cutoff logic is no longer valid since we don't set the cutoff and don't have UI to change it. i should really just remove it.